.github/workflows/deploy-branch.yml

```diff
@@ -0,0 +1,46 @@
+name: Deploy-dev
+
+on:
+  pull_request:
+jobs:
+  deploy-dev:
+    runs-on: ubuntu-latest
+    container: tarantool/doc-builder:v1
+    env:
+      AWS_ACCESS_KEY_ID: ${{secrets.AWS_ACCESS_KEY_ID}}
+      AWS_SECRET_ACCESS_KEY: ${{secrets.AWS_SECRET_ACCESS_KEY}}
+      AWS_DEFAULT_REGION: ${{secrets.AWS_DEFAULT_REGION}}
+      S3_ENDPOINT_URL: ${{secrets.S3_ENDPOINT_URL}}
+      S3_UPLOAD_PATH: ${{secrets.S3_UPLOAD_PATH}}
+      TARANTOOL_UPDATE_KEY: ${{secrets.TARANTOOL_UPDATE_KEY}}
+      TARANTOOL_UPDATE_URL: ${{secrets.TARANTOOL_DEV_UPDATE_URL}}
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Start dev server deployment
+        uses: bobheadxi/deployments@v0.5.2
+        id: deployment
+        with:
+          step: start
+          token: ${{secrets.GITHUB_TOKEN}}
+          env: release
+          ref: ${{github.head_ref}}
+
+      - run: cmake .
+      - run: make pull-modules
+      - run: make build-modules
+      - run: make json
+      - run: make json-ru
+
+      - name: Set branch name from source branch
+        run: echo "BRANCH_NAME=${GITHUB_HEAD_REF#refs/heads/}" >> $GITHUB_ENV
+      - run: bash upload_output.sh
```

Comment:
Will it remove the files from S3 when this deployment isn't needed anymore? How do we remove deployments?

Reply:
No, will create a new workflow for it later